torch/testing/_internal/opinfo/refs.py

```diff
@@ -113,6 +113,23 @@ def __init__(
         self.supports_nvfuser = supports_nvfuser
         assert isinstance(self.torch_opinfo, OpInfo)
 
+        def test_ref_export(name):
+            # make sure name is exported via __all__
+            import torch
+
+            lst = name.split(".")
+            mod_name = lst[:-1]
+            func_name = lst[-1]
+
+            mod = torch
+            for i in range(len(mod_name)):
+                mod = getattr(mod, mod_name[i])
+
+            if hasattr(mod, "__all__"):
```

Comment:
What's a case of a module that doesn't have the `__all__` attribute and why?

Reply:
ops.nvprims.var_mean, see above